src/app/modules/shared-components/components/page-navigator/page-navigator.component.html

```diff
@@ -1,6 +1,12 @@
 <div class="page-nav">
+  <alg-watch-button
+    [route]="route"
+    [startWatchGroupInfo]="startWatchGroupInfo"
+    *ngIf="allowWatching && startWatchGroupInfo"
```

Comment:
Actually, make the watch button follow the current content, it will cleaner for 2 generic places where they are used.

Reply:
Fixed